page/sass/commit_funds.scss

```diff
@@ -47,13 +47,14 @@
       }
       .section-seperator{
         border-bottom: 1px $gray-extra-light solid;
-        width: 407px;
+        width: 90%;
         margin-left: 50px;
```

Comment:
Can you create new variable for this number in the variable.scss file?

Reply:
This will be taken off